### PR TITLE
Add labels to input elements

### LINKED ISSUE #1215: Add labels to every input field used as example [73f2c2]
As reported by @daniel-montalvo in https://github.com/act-rules/act-rules.github.io/pull/1189#discussion_r388804284 we have several test cases with `input` elements without a corresponding label. This is normally in unrelated rules (e.g., most examples of [`autocomplete` attribute has valid value](https://act-rules.github.io/rules/73f2c2)).

Should we add a label to these?
The absence of a label does not make the (passed) examples fail the SC, but it is still a bad practice…

### PR DESCRIPTION
**Pull request by Jym77**
Add labels to most `input` elements used in examples, as per CG decision.

I've left out `input` in the following rules:
* https://act-rules.github.io/rules/6cfa84
* https://act-rules.github.io/rules/e6952f
* https://act-rules.github.io/rules/674b10
* https://act-rules.github.io/rules/4e8ab6
because they are all extremely bare minimum `input`s.

I've also tried to avoid adding too many labels (i.e. in case where no label is needed for the example to "work"), please pay special attention in checking that it is the case 😄 

Closes issue(s):

- closes #1215

Need for Final Call:
This will require a 1 week Final Call (changing examples)

---

### **When creating PR:**

- [X] Make sure you're requesting to **pull a branch** (right side) to the `develop` branch (left side).

### **After creating PR:**

- [X] Add yourself (and co-authors) as "Assignees" for PR.
- [X] Add label to indicate if it's a `Rule`, `Definition` or `Chore`.
- [X] [Link the PR to any issue it solves](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue). This will be done automatically by [referencing the issue](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword) at the top of this comment in the indicated place.
- [X] Optionally request feedback from anyone in particular by assigning them as "Reviewers".

### **When merging a PR:**

- [ ] Close any issue that the PR resolves. This will happen automatically upon merging if the PR was correctly linked to the issue, e.g. by referencing the issue at the top of this comment.

## How to Review And Approve

- Go to the “Files changed” tab
- Here you will have the option to leave comments on different lines.
- Once the review is completed, find the “Review changes” button in the top right, select “Approve” (if you are really confident in the rule) or "Request changes" and click “Submit review”.
- Make sure to also review the proposed Final Call period. In case of disagreement, the longer period wins.

### CODE REVIEW

**Comment by Jym77** [comment]
Final Call ends on June 16.

File: _rules/aria-attr-defined-5f99a7.md [hunk 1]
```diff
@@ -71,7 +71,10 @@ This `div` element with a role of `alert` has an `aria-live` attribute which is
 This `input` element with a role of `spinbutton` has multiple `aria-*` attributes specified which are all defined in [WAI-ARIA Specifications][].
 
 ```html
-<input role="spinbutton" aria-valuemax="100" aria-valuemin="0" aria-valuenow="25" type="number" value="25" />
+<label
+	>Enter a number between 0 and 100:
```
Comment (EmmaJP) [inline _rules/aria-attr-defined-5f99a7.md:75]: It seems add to have a new line in the middle of a tag. Minor typo, but should probably be fixed.

File: _rules/autocomplete-valid-value-73f2c2.md [hunk 2]
```diff
@@ -100,55 +100,55 @@ Autocomplete term, only valid for textarea.
 Two autocomplete terms.
 
 ```html
-<input autocomplete="Work EMail" />
+<label>Work email<input autocomplete="Work EMail"/></label>
```
Comment (EmmaJP) [inline _rules/autocomplete-valid-value-73f2c2.md:103]: Does this autocomplete value reference something? Its unusual camel casing in my experience, so doesn't read like a variable name, or something legit for an autocomplete value.

https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/autocomplete

File: _rules/aria-attr-defined-5f99a7.md (hunk 1 above)
Comment (Jym77) [reply _rules/aria-attr-defined-5f99a7.md]: That prettier being somewhat aggressive and cuting the line due to the long `input` element…
I've replaced this with a `label for=…` to make the result a bit more "logical".

File: _rules/autocomplete-valid-value-73f2c2.md (hunk 2 above)
Comment (Jym77) [reply _rules/autocomplete-valid-value-73f2c2.md]: autocomplete tokens are match case-insensitive, so this is technically OK.
I've nonetheless corrected it as it is indeed a bit confusing and definitely not what the example illustrates.

**Comment by Jym77** [comment]
Final Call has ended. Merging.

File: _rules/autocomplete-valid-value-73f2c2.md (hunk 2 above)
Comment (EmmaJP) [reply _rules/autocomplete-valid-value-73f2c2.md]: Why is it not all lower case like the other examples of autocomplete?

Comment (Jym77) [reply _rules/autocomplete-valid-value-73f2c2.md]: That, I don't know and deem it an unrelated issue that need (or not) to be tackled in another PR…

**Pull request merged**